public/app.js

```diff
@@ -99,6 +87,19 @@ const controllers = {
         alertBox.innerHTML = `Successfully created wilder ${wilder.firstName} (${wilder.id})`
       })
     })
+    const navbarDejaInscrit = document.getElementById("navbarMenu")
+    navbarDejaInscrit.innerHTML = `<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
it is better to indent it like this: 
`el.innerHtml = 
  <nav>
     ...
  </nav>`